src/Apps/Sell/SellFlowContext.tsx

```diff
@@ -1,33 +1,50 @@
 import { useToasts } from "@artsy/palette"
-import { CreateSubmissionMutationInput } from "__generated__/CreateConsignSubmissionMutation.graphql"
+import {
+  ConsignmentSubmissionStateAggregation,
+  CreateSubmissionMutationInput,
+} from "__generated__/CreateConsignSubmissionMutation.graphql"
 import { UpdateSubmissionMutationInput } from "__generated__/UpdateConsignSubmissionMutation.graphql"
 import { useCreateSubmissionMutation$data } from "__generated__/useCreateSubmissionMutation.graphql"
 import { useUpdateSubmissionMutation$data } from "__generated__/useUpdateSubmissionMutation.graphql"
 import { useSubmissionTracking } from "Apps/Sell/Hooks/useSubmissionTracking"
 import { useCreateSubmission } from "Apps/Sell/Mutations/useCreateSubmission"
 import { useUpdateSubmission } from "Apps/Sell/Mutations/useUpdateSubmission"
-import { createContext, useContext, useEffect, useState } from "react"
+import { createContext, useContext, useEffect, useMemo, useState } from "react"
+import { useFeatureFlag } from "System/Hooks/useFeatureFlag"
 import { useRouter } from "System/Hooks/useRouter"
 import { useCursor } from "use-cursor"
 import createLogger from "Utils/logger"
 
 const logger = createLogger("SellFlowContext.tsx")
 
-export const STEPS = [
+const BASIC_STEPS = [
```

Comment:
I'll have a look if we can use [WorkflowEngine](https://github.com/artsy/force/blob/760cfe1924c45975ab23fdf164353271323c0fc9/src/Utils/WorkflowEngine.tsx) here.